src/terms.ml

```diff
@@ -94,6 +94,8 @@ type term =
   (** The symbol may have a definition or rewriting rules (but NOT both). *)
   | Injec
   (** Same as [Defin], but the symbol is considered to be injective. *)
+  | Priva
```

Comment:
There is no uniformity in the names of constructors. In one file, this is const, inj or prv. In another one, this is Injec or Priva. We should uniformize all this. Why not simply write Injective and Private everywhere?

Reply:
Should I modify it in this PR?